ui/documents/semantic-ui/js/documents/search/ResultsListItem.jsx

```diff
@@ -252,11 +252,19 @@ export const ResultsListItemComponent = ({
                 <ItemSidebarIcons rights={rights} accessStatus={accessRights} />
               </Grid.Column>
               <Grid.Column className="results-list item-main">
-                <ItemHeader
-                  title={title}
-                  searchUrl={searchAppConfig.ui_endpoint}
-                  selfLink={result.links.self_html}
-                />
+                <div className="justify-space-between flex">
+                  <ItemHeader
+                    title={title}
+                    searchUrl={searchAppConfig.ui_endpoint}
+                    selfLink={result.links.self_html}
+                  />
+                  <div>
```

Comment:
whole div should be rendered only if there is both result.state and accessRights